src/app/store/actions/ReferenceData.actions.spec.ts

```diff
@@ -45,3 +47,13 @@ describe('LoadTestStationsSuccess', () => {
     });
   });
 });
+
+describe('LoadTestTypeCategoriesSuccess', () => {
+  it('should have the correct action type & test type categories in taxonomy payload when LoadTestTypeCategoriesSuccess() action is instantiated', () => {
```

Comment:
rephrase test titles for actions. `it should create LoadTestTypeCategoriesSuccess action`. Action should create instances. Please lets update all necessary